oneflow/core/common/data_type.cpp

```diff
@@ -130,6 +130,54 @@ size_t GetSizeOfDataType(DataType data_type) {
   }
 }
 
+int64_t GetIntMaxVal(DataType datatype) {
+#define SWITCH_INT_TYPE(cpp_type, of_datatype) \
+  case of_datatype: return static_cast<int64_t>(GetMaxVal<DataTypeToType<of_datatype>>());
+  switch (datatype) {
```

Comment:
```suggestion
  case of_datatype: return static_cast<int64_t>(GetMaxVal<DataTypeToType<of_datatype>>());

  switch (datatype) {
```

这里换下行是不是好一点。好像上面那个 cpp 文件也有类似的宏定义和代码哎太紧，一眼不容易分开的情况。

Reply:
已改